src/Model/Filter/Boolean.php

```diff
@@ -0,0 +1,40 @@
+<?php
+namespace Search\Model\Filter;
+
+class Boolean extends Base
+{
+
+    /**
+     * Default configuration.
+     *
+     * @var array
+     */
+    protected $_defaultConfig = [
+        'truthy' => [1, true, '1', 'true',  'yes', 'on'],
```

Comment:
It makes sense either way - only then you could for example refactor without danger of forgetting things. But never mind that now.

All right then.